templates/members/member_bank.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+
+{% block hos_content %}
+    <p>
+        <h1>Sepa Export</h1>
+        <form method="post" action="/member/bankexport/">
+            {% csrf_token %}
+            <button type="submit">Export SEPA.xml</button> (warning: will update mandate date on new mandates/members)
```

Comment:
What consequence have the mandate date update?

Reply:
The mandate date is currently used to check if this is the first or a recurring collection [(and updates the field)](https://github.com/Metalab/mos/pull/111/files#diff-3a725d11ad1f1306ba69f7c711601abac78eeb5e2bb0acf79c6f846c649d18cdR175), which must be specified in SEPA. (Also in elba, but elba didn't care about it)

I'll rephrase the warning.